--- archived
+++ fresh
@@ -325,17 +325,17 @@
 								   
 									<tr>
 							  		 	<td> Department of Regulatory Agencies </td>
-							  		 	<td> Division of Insurance </td>
-							  		 	<td> 
-							  		 	
-											<a href="/CCR/Upload/NoticeOfRulemaking/ProposedRuleAttach2026-00337.docx" target="_blank">
-											<span style="text-decoration: underline;">LIFE, ACCIDENT AND HEALTH, Series 4-2 Accident and Health (General)</span></a>
-										
-										
-										
-
-							  		 	</td>
-							  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00337')" ><span style="text-decoration: underline;">09/02/2026 03:00 PM </span></a> </td>
+							  		 	<td> Public Utilities Commission </td>
+							  		 	<td> 
+							  		 	
+											<a href="/CCR/Upload/NoticeOfRulemaking/ProposedRuleAttach2026-00329.docx" target="_blank">
+											<span style="text-decoration: underline;">RULES REGULATING RAILROADS, RAIL FIXED GUIDEWAYS, TRANSPORTATION BY RAIL, AND RAIL CROSSINGS</span></a>
+										
+										
+										
+
+							  		 	</td>
+							  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00329')" ><span style="text-decoration: underline;">10/08/2026 11:00 AM </span></a> </td>
 									</tr>
 								   
 									<tr>
@@ -343,21 +343,6 @@
 							  		 	<td> Public Utilities Commission </td>
 							  		 	<td> 
 							  		 	
-											<a href="/CCR/Upload/NoticeOfRulemaking/ProposedRuleAttach2026-00329.docx" target="_blank">
-											<span style="text-decoration: underline;">RULES REGULATING RAILROADS, RAIL FIXED GUIDEWAYS, TRANSPORTATION BY RAIL, AND RAIL CROSSINGS</span></a>
-										
-										
-										
-
-							  		 	</td>
-							  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00329')" ><span style="text-decoration: underline;">10/08/2026 11:00 AM </span></a> </td>
-									</tr>
-								   
-									<tr>
-							  		 	<td> Department of Regulatory Agencies </td>
-							  		 	<td> Public Utilities Commission </td>
-							  		 	<td> 
-							  		 	
 											<a href="/CCR/Upload/NoticeOfRulemaking/ProposedRuleAttach2026-00330.docx" target="_blank">
 											<span style="text-decoration: underline;">RULES FOR SAFETY PROGRAM STANDARDS FOR RAIL FIXED GUIDEWAY SYSTEMS [Repealed eff. 04/01/2006]</span></a>
 										
@@ -516,21 +501,6 @@
 
 							  		 	</td>
 							  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00331')" ><span style="text-decoration: underline;">09/01/2026 11:00 AM </span></a> </td>
-									</tr>
-								   
-									<tr>
-							  		 	<td> Department of Health Care Policy and Financing </td>
-							  		 	<td> Medical Services Board (Volume 8; Medical Assistance, Children&#39;s Health Plan) </td>
-							  		 	<td> 
-							  		 	
-											<a href="/CCR/Upload/NoticeOfRulemaking/ProposedRuleAttach2026-00328.docx" target="_blank">
-											<span style="text-decoration: underline;">MEDICAL ASSISTANCE - STATEMENTS OF BASIS AND PURPOSE AND RULE HISTORY</span></a>
-										
-										
-										
-
-							  		 	</td>
-							  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00328')" ><span style="text-decoration: underline;">09/11/2026 09:00 AM </span></a> </td>
 									</tr>
 								   
 									<tr>
@@ -1224,12 +1194,6 @@
 										</tr>
 									   
 										<tr>
-								  		 	<td style="text-align:left;"> Division of Insurance </td>
-								  		 	<td> LIFE, ACCIDENT AND HEALTH, Series 4-2 Accident and Health (General) </td>
-								  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00337')" ><span style="text-decoration: underline;">09/02/2026 03:00 PM </span></a> </td>
-										</tr>
-									   
-										<tr>
 								  		 	<td style="text-align:left;"> Public Utilities Commission </td>
 								  		 	<td> RULES REGULATING RAILROADS, RAIL FIXED GUIDEWAYS, TRANSPORTATION BY RAIL, AND RAIL CROSSINGS </td>
 								  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00329')" ><span style="text-decoration: underline;">10/08/2026 11:00 AM </span></a> </td>
@@ -1299,12 +1263,6 @@
 								  		 	<td style="text-align:left;"> Division of Fire Prevention and Control </td>
 								  		 	<td> COLORADO WILDFIRE RESILIENT HOMES GRANT PROGRAM </td>
 								  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00331')" ><span style="text-decoration: underline;">09/01/2026 11:00 AM </span></a> </td>
-										</tr>
-									   
-										<tr>
-								  		 	<td style="text-align:left;"> Medical Services Board (Volume 8; Medical Assistance, Children&#39;s Health Plan) </td>
-								  		 	<td> MEDICAL ASSISTANCE - STATEMENTS OF BASIS AND PURPOSE AND RULE HISTORY </td>
-								  		 	<td> <a href="javascript:void(0)" onclick="OpenDetailsWindow('2026-00328')" ><span style="text-decoration: underline;">09/11/2026 09:00 AM </span></a> </td>
 										</tr>
 									   
 										<tr>
@@ -1370,5 +1328,5 @@
 </td></tr>
 </table>
 	
-<script>(function(){function c(){var b=a.contentDocument||(a.contentWindow&&a.contentWindow.document);if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'a38a0310dda086e1',t:'MTc4ODk5Njk1Mw=='};var a=document.createElement('script');a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script></body>
+<script>(function(){function c(){var b=a.contentDocument||(a.contentWindow&&a.contentWindow.document);if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'a3a28242ed49e673',t:'MTc4OTI1MzgyMQ=='};var a=document.createElement('script');a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script></body>
 </html>